dash/include/dash/Meta.h

```diff
@@ -90,6 +90,19 @@ struct is_container_compatible :
          >
 { };
 
+/**
+ * Type trait indicating whether a type can be used for global atomic
+ * operations.
+ */
+template <typename T>
+struct is_atomic_compatible
+: public std::integral_constant<
+           bool,
+              dash::is_container_compatible<T>::value
+           && sizeof(T) <= sizeof(std::size_t)
+         >
```

Comment:
... So: yes, eventually the atomic type traits should depend on types supported by atomic operations in DART or even split into `dash::is_atomic_compatible` and `dash::is_global_atomic_compatible`, but for now the definition in this PR should be correct.